PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -293,7 +289,7 @@ if [ "${T2wPresent}" = "YES" ] ; then
 	fi
 	BiasFieldComputed=false
 	#Reduce memory usage by smoothing on downsampled mesh (match the gifti version by using the first lowresmesh)
-	LowResMesh="${LowResMeshesArray[0]}"
+	LowResMesh="${LowResMeshesArray[0]}"		
 	# myelin map only loop
 	for MyelinMap in MyelinMap SmoothedMyelinMap ; do
 		if [ "$BiasFieldComputed" = false ]; then
```

Comment:
I don't see a comment that I thought I made here, so making it again...this loop is hardcoded to always execute twice, and the first time it will do this `= false` section, and the second time it will do the `else`.  Therefore, all of this `if` statement can instead be translated into appropriate unconditional commands before this loop.

Reply:
I see. Since the BC operation for `MyelinMap` is already part of the module `MyelinMap_BC.sh`, there will still be a `if` condition to allow single BC operation (`Var-bias`) on the other types of myelin maps, such as `SmoothedMyelinMap`.